src/ReactARTFiber.js

```diff
@@ -0,0 +1,496 @@
+/**
+ * Copyright (c) 2013-present Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactART
+ */
+
+'use strict';
+
+require('art/modes/current').setCurrent(
+  // Change to 'art/modes/dom' for easier debugging via SVG
+  require('art/modes/fast-noSideEffects'),
+);
+
+const Mode = require('art/modes/current');
+const Transform = require('art/core/transform');
+const invariant = require('fbjs/lib/invariant');
+const React = require('react');
+const ReactFiberReconciler = require('react-dom/lib/ReactFiberReconciler');
```

Comment:
Oh! Gotcha. 😄

I tried using `npm link` at first but ran into problems with Babel trying to reprocess the dist for 'react' and 'react-dom'. Seems like the symlink created by `npm link` caused the exclude for 'node_modules' not to work for those packages. So I gave up on that after a while.

What I ended up doing was a bit of a hack- but it worked, so...

I checked out React master and copied the built versions of 'react' and 'react-dom' into the `node_modules` folder of 'react-art'. So basically...

```
cd path/to/react
cp -r ./build/packages/react path/to/your/project/node_modules/
cp -r ./build/packages/react-dom path/to/your/project/node_modules/
```